GameServerLib/ObjectManager.cs

```diff
@@ -245,9 +245,10 @@ void LateUpdate(IGameObject obj, float diff)
         {
             // Destroy any missiles which are targeting an untargetable unit.
             // TODO: Verify if this should apply to SpellSector.
+            // TODO: Should it be moved to SpellMissile.Update?
```

Comment:
Yes, move to SpellMissile.Update.
It already has a check for `HasTarget`.

Reply:
Done, I suppose?